_usage/limitations-of-babelfish.md

```diff
@@ -370,4 +371,4 @@ The following escape hatches exist:
 | `babelfishpg_tsql.escape_hatch_storage_on_partition` | Controls Babelfish behavior related to the `ON partition_scheme` column clause when defining partitioning. Babelfish currently doesn't implement partitioning. | strict |
 | `babelfishpg_tsql.escape_hatch_storage_options` | Escape hatch on any storage option used in `CREATE`/`ALTER` for `DATABASE`, `TABLE` and `INDEX`.  This includes the clauses `(LOG) ON`, `TEXTIMAGE_ON`, `FILESTREAM_ON` that define storage locations (partitions, filegroups) for tables, indexes, and constraints, and also for a database. This escape hatch setting applies to all of these clauses (including `ON PRIMARY` and `ON DEFAULT`). The exception is when a partition is specified for a table or index with `ON partition_scheme (column)`. | ignore |
 | `babelfishpg_tsql.escape_hatch_table_hints` | Controls the behavior of table hints specified using the `WITH (...)` clause. | ignore |
-| `babelfishpg_tsql.escape_hatch_unique_constraint` | Controls Babelfish behavior toward unsupported session-level `SET` statements. | strict |
+| `babelfishpg_tsql.escape_hatch_unique_constraint` | Controls Babelfish behavior when creating a unique index or constraint on a nullable column. | strict |
```

Comment:
From tech perspective, as long as it refers to both unique index or unique constraint, looks good to me.